src/components/OrbitDBStoreDisplay.tsx

```diff
@@ -118,12 +108,7 @@ const OrbitDBEventStoreDisplay: React.FC<{
    * @param operationsLog The operations log used to reconstruct the database state.
    */
   function reconstructData(operationsLog: OperationsLog): Array<any> {
-
-    let index = new EventIndex();
-    let ipfsLog = operationsLog.getInnerLog();
-    index.updateIndex(ipfsLog);
-    let result = index.get();
-    return result;
+    return nodeProvider.reconstructData(operationsLog);
```

Comment:
Should we simply not have this method (if all it does is wrap the Node Provider method?)

Reply:
Good point